interface-definitions/include/firewall/rule-log-options.xml.i

```diff
@@ -0,0 +1,88 @@
+<!-- include start from firewall/rule-log-options.xml.i -->
+<node name="log-options">
+   <properties>
+    <help>Log options</help>
+  </properties>
+  <children>
+    <leafNode name="group">
+      <properties>
+        <help>Set log group. Log must be enable.</help>
```

Comment:
I think we can skip this redundant string "Log must be enable", it dhould be displayed during `commit` if log was not configured.